test/google.coffee

```diff
@@ -517,11 +523,25 @@ describe 'OrgUnitProvisioning', ()->
 
   it 'uses cache for OU lookup', (done) ->
     org_unit = ['Students']
-    cache = {'/Students':1, '/Students/Schoolname':1}
-    @ou.findOrCreate 'abcdef', org_unit, cache, (err, parent, cache) ->
+    first_ou =
+      kind: 'admin#directory#orgUnit'
+      name: 'Students'
+      orgUnitPath: '/Students'
+      orgUnitId: 'ou_id1'
+      parentOrgUnitPath: '/'
+    second_ou =
+      kind: 'admin#directory#orgUnit'
+      name: 'Schoolname'
+      orgUnitPath: '/Students/SchoolName'
+      orgUnitId: 'ou_id2'
+      parentOrgUnitPath: '/Students'
+    cache =
+      '/Students': first_ou
```

Comment:
Nit, but one possible improvement is to use `first_ou.orgUnitPath`, fewer typo possibilities.